courses/free-courses-te.md

```diff
@@ -1,39 +1,222 @@
+* [Cloud Computing](#cloud-computing)
+* [Compiler Design](#compiler-design)
+* [CSS](#css) 
+* [Database Management Systems](#database-management-systems)
+* [Data Science](#data-science)
+* [Data Structures](#data-structures)
+* [Deep Learning](#deep-learning)
+* [Ethical Hacking](#ethical-hacking)
+* [ExpressJS](#expressjs)
+* [HTML](#html)
 * [Java](#java)
 * [Javascript](#javascript)
+* [Laravel](#laravel)
+* [Machine Learning](#machine-learning)
+* [MongoDB](#mongodb)
+* [NextJS](#nextjs)
+* [NodeJS](#nodejs)
+* [PHP](#php)
 * [Python](#python)
 * [R](#r)
+* [ReactJS](#reactjs)
+* [SQL](#sql)
+* [Tailwind CSS](#tailwind-css)
+* [VueJs](#vuejs)
+
+
+### Android
+
+* [Android Development course in 2 Hours in Telugu](https://www.youtube.com/watch?v=7ZLKpN8vXLo) - Python Life
+* [Android App Development Tutorials For Beginners in Telugu Android Studio](https://www.youtube.com/playlist?list=PLeCpoxUq3EhmgHp7KI-Ih3X2PcNYRtC5y) - Sai Gopi
+
+
+### Angular
+
+* [Angular In Telugu | ANGULAR 10 IN TELUGU |angular in telugu|ANGULAR INTRODUCTION FOR BEGINNERS(2021)](https://www.youtube.com/watch?v=9MxS8oNlnMM) - TeluguTechSteps
+* [Angular In Telugu](https://www.youtube.com/playlist?list=PLO7Oa5iXf4QhtPXkaNX05qhGQSKFsvAF7) - TeluguTechSteps
+
+
+### Artificial Intelligence
+
+* [Artificial Intelligence 12 Hours Course In Telugu](https://www.youtube.com/watch?v=fwMMBaXIpqQ) - Python Life
+
+
+### Automata Theory
+
+* [AUTOMATA THEORY](https://www.youtube.com/playlist?list=PLLOxZwkBK52CTVrHjYa7-SpXlEtef1TqL) - Sundeep Saradhi Kanthety
+
+
+### Bash and Shell
+
+* [Bash Scripting in Telugu](https://www.youtube.com/playlist?list=PLd8alL65M1GYJOLGK312G1qDv-Tv9aBbs) - Trie Tree Technologies
+* [Shell Scripting full course In Telugu by 7Hills | Linux In Telugu | Bash scripting | programming](https://www.youtube.com/watch?v=Duq5MtBEChc) - 7 Hills 
+
+
+### Bootstrap
+
+* [BOOTSTRAP INTRODUCTION IN TELUGU](https://www.youtube.com/watch?v=1MrR6eTQYgI) - TeluguTechSteps
+* [Bootstrap Tutorials in Telugu by Kotha Abhishek](https://www.youtube.com/playlist?list=PLv_sM9ZH4RUU9Yopth1p1DbEu1ebr1vTb) - Chintu Tutorials
 
 
 ### C
 
-* [C Language Tutorials](https://www.youtube.com/playlist?list=PLC2mgeYbYNm-n8Iz-_3MuNsJFzr0UlGUu) - Telugu Computer World (YouTube)
+* [C Language in Telugu - Complete Tutorial in 12 Hours](https://www.youtube.com/watch?v=HdvRHC5TiwE) - Telugu Computer World
+* [C Language Tutorial](https://www.youtube.com/playlist?list=PL3KKfF5A0sSKZutcrUiTCQDX0hn7Tw61E) - Telugu Scit Tutorials
+
+
+### C++
+
+* [C++ Complete Tutorial](https://www.youtube.com/watch?v=uZBXKmQH5u8) - Telugu Computer World
+* [C++ Programming in Telugu - Complete Tutorial in 14 Hours](https://www.youtube.com/playlist?list=PLC2mgeYbYNm9keJjsA95jKa4EUVLd7mQP) - Telugu Computer World
+
+
+### Cloud Computing
+
+* [Introduction to Cloud Computing in Telugu | Cloud Computing Tutorial in Telugu | Edureka Telugu](https://www.youtube.com/watch?v=hcaROQRJM6k) - edureka! Telugu
```

Comment:
This is only for 19 mins. We dont allow tutorials < 1hr in length. Please remove